app.ts

```diff
@@ -119,6 +119,12 @@ setInterval(() => {
           },
           { merge: true }
         )
+        db.collection("members")
+        doc("sourceStatus")
```

Comment:
Isn't this supposed to be .doc("sourceStatus") ? 
Looks like there's a missing dot

Reply:
Yes , Will add that 👍